crates/matrix-sdk/src/widget_api/messages/mod.rs

```diff
@@ -31,12 +31,15 @@ impl From<ToWidgetMessage> for Message {
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct Header {
+    #[serde(rename="requestId")]
     pub request_id: String,
+    #[serde(rename="widgetId")]
     pub widget_id: String,
 }
```

Comment:
We could instead add a `#[serde(rename_all = "camelCase")]` in front of a `Header` definition, so that we don't need to do it on per-field basis.